tests.py

```diff
@@ -27,9 +30,14 @@ def run_vsm(self, name, input_data, expected_output, use_initial=True):
 
         process = Popen(cmd, stdin=PIPE, stdout=PIPE)
 
-        output, _ = process.communicate(input=input_data.encode(), timeout=2)
-
-        self.assertEqual(output.decode() , expected_output)
+        output, _ = process.communicate(input=input_data.encode(), timeout=5)
```

Comment:
You don't need to print the dict directly, you can use a for loop, say, and print the entries sorted.